pkg/operation/botanist/component/extensions/operatingsystemconfig/operatingsystemconfig.go

```diff
@@ -66,6 +66,8 @@ type Interface interface {
 	DeleteStaleResources(context.Context) error
 	// WaitCleanupStaleResources waits until all unused OperatingSystemConfig resources are cleaned up.
 	WaitCleanupStaleResources(context.Context) error
+	// SetAPIServerAdress sets the APIServerURL value.
+	SetAPIServerAdress(string)
```

Comment:
There is a typo in the name (`Adress` with single d), but perhaps it's better to use `URL` here to be consistent with the other names?
```suggestion
	SetAPIServerURL(string)
```

Reply:
Good catch, thank you. Fixed in a new commit.
/squash